greedybear/settings.py

```diff
@@ -15,6 +15,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get("DEBUG", False) == "True"
 
+# DB Backup Config
+# Backup Interval in minutes. Default: 1440 min = 1 day.
+DB_BACKUP_FREQUENCY = int(os.environ.get("DB_BACKUP_FREQUENCY", 1440)) or 1440
```

Comment:
have you tried to customize this value? cause this does not work.

you have to leverage the env file that is loaded by the application

Reply:
Ok Sir. I'll try debugging it.